### ISSUE TITLE
v1.1.0 is broken in go1.16

### ISSUE THREAD

**Issue by srimaln91** [twpayne/go-polyline#7]
Hi,

Thank you for this awesome project.

I'm getting the following error when importing the package in Go 1.16

```bash
../../go/pkg/mod/github.com/twpayne/go-polyline@v1.1.0/polyline.go:93:12: undefined: math.MaxUint
../../go/pkg/mod/github.com/twpayne/go-polyline@v1.1.0/polyline.go:94:10: undefined: math.MinInt
```
It seems there are some breaking changes in math package in recent Go versions.
https://go-review.googlesource.com/c/go/+/317911/

Please let me know if you want any contribution from my end.

Thank you again.

**Comment by twpayne** [comment]
Thanks for reporting this. #8 fixes it, and I've tagged `v1.1.1` so you can `go get` the fixed version.

**Comment by srimaln91** [comment]
Thank you @twpayne 🙂
